README.md

```diff
@@ -270,7 +270,8 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 },
 
                 // The path to the TLS certificates directory.
-                // Used for HTTPS connections.
+                // Used for client HTTPS connections to the OpenID Provider.
```

Comment:
The use of the word "client" here is confusing to me for some reason. I'm guessing it's referring to the HTTP API server acting as a client to the OpenID Provider? Omitting the word might reduce confusion, or it might help to add more words. Or... it's just me. :)

Reply:
I think it is ... correct?

we're the 'caller/client', not the 'server' for the https connection...

Reply:
No, I was also wondering if that phrase needed a tweak.

How about ...
```diff
-// The path to the TLS certificates directory.
-// Used for client HTTPS connections to the OpenID Provider.
+// The path to the TLS certificates directory used to establish
+// secure connections between the HTTP API and the OpenID provider.
```

Reply:
I threw in client because of the discussion & note this morning: https://github.com/irods/irods_client_http_api/issues/243#issuecomment-2455151700

It is confusing as is, though. It's supposed to refer to the fact that the TLS certificates are used by the 'client' (us) in a connection to a server 'openid'.

Reply:
Okay, then I think that it should be made clearer that the word "client" is referring to the HTTP API server and not the client of the HTTP API server. Here are a few suggestions, in order of preference:
1. Used for HTTPS connections between the HTTP API server and the OpenID Provider.
2. Used for client HTTPS connections between the HTTP API server and the OpenID Provider.
3. Used for HTTPS connections to the OpenID Provider.
4. Used for client (i.e. the HTTP API server) HTTPS connections to the OpenID Provider.

Reply:
Whoops, got sniped. I like @korydraughn's suggestion as well.

Reply:
agreed, vote for kory.

Reply:
Done.